jenkins/scripts/VersionSelectorScript.groovy

```diff
@@ -100,7 +100,7 @@ def buildExclusions = [
 
   // Shared libs docker containers -------------------------
   [ /ubi81_sharedlibs/,               anyType,     lt(13)  ],
-  [ /sharedlibs_openssl300/,          anyType,     lt(15)  ],
+  [ /sharedlibs_openssl3/,            anyType,     lt(15)  ],
```

Comment:
This should match both `sharedlibs_openssl300` (the current label) and `sharedlibs_openssl3` (I'm planning to rename after landing). The general idea being that OpenSSL 3 and later now follows semver and genericizing the variable names/labels should mean less places to update in the future.